idaes/models_extra/power_generation/unit_models/helm/tests/test_turbine_multistage.py

```diff
@@ -165,11 +176,11 @@ def test_initialize_calc_cf():
 
     # Set the inlet of the ip section, which is disconnected
     # here to insert reheater
-    p = 7.802e06
+    p = 1.4e06
     hin = pyo.value(iapws95.htpx(T=880 * pyo.units.K, P=p * pyo.units.Pa))
     m.fs.turb.ip_stages[1].inlet.enth_mol[0].value = hin
     # m.fs.turb.ip_stages[1].inlet.flow_mol[0].value = 25220.0
-    m.fs.turb.ip_stages[1].inlet.pressure[0].value = p
+    # m.fs.turb.ip_stages[1].inlet.pressure[0].value = p
```

Comment:
Do we need these commented lines? If so, a reason might help for future reference.

Reply:
Why did the `p` value change?

Reply:
The original was just an initial guess for initialization.  I set the inlet stage pressure ratio to 0.6, and in this case use that to calculate the inlet stage flow coefficient.  So the original pressure was really far off.  I think I did the original test in a hurry and despite the bad initialization setup, it passed and I didn't give it any more thought.

Reply:
Okay, thanks for clarifying. I trust your judgement here.